packages/react/src/components/Masthead/MastheadTopNav.js

```diff
@@ -21,36 +25,62 @@ import {
  * @returns {*} Masthead top nav component
  */
 const MastheadTopNav = ({ navigation, ...topNavProps }) => {
+  /**
+   * Event data for CoreMetrics
+   *
+   * @param {string} title title of the menu item
+   * @param {number} key key for id of menu item
+   * @param {string} type type of element
+   * @returns {object} event fields
+   */
+  const menuItemEventData = (title, key, type) => ({
+    type: 'element',
+    primaryCategory: 'MASTHEAD',
+    eventName: 'HOVER',
+    executionPath: `masthead__l0-nav--nav-${key}`,
+    execPathReturnCode: type,
+    targetTitle: title,
```

Comment:
Do we need any conditionals/validation for some of these values in case the information is corrupt?